docs/debugging-python-lambda-functions.md

```diff
@@ -0,0 +1,92 @@
+# Debugging Python lambda functions
+
```

Comment:
Consider an intro sentence, like
```
These instructions outline how you can start a Sam Application locally using the SAM CLI, and attach the VS Code debugger to it.
```

I feel this is important to clarify that we're setting up to attach, compared to more involved instructions that would perform a more integrated debug experience.